### PR TITLE
monero-cli: Add version 0.14.1.0

### PR DESCRIPTION
**Pull request by niheaven**
For 32bit

### CODE REVIEW

File: bucket/monero-cli.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            "64bit": {
+                "url": "https://dlsrc.getmonero.org/cli/monero-win-x64-v$version.zip"
+            },
+            "32bit": {
+                "url": "https://dlsrc.getmonero.org/cli/monero-win-x86-v$version.zip"
+            }
+        },
+        "hash": {
+            "url": "https://getmonero.org/downloads/hashes.txt",
+            "find": "$basename.*([a-fA-F0-9]{64})"
```
Comment (Ash258) [inline bucket/monero-cli.json]:
```suggestion
            "regex": "$basename,\\s+$sha256"
```

Comment (niheaven) [reply bucket/monero-cli.json]: Why `,\\s+`?

Comment (Ash258) [reply bucket/monero-cli.json]: `monero-win-x64-v0.14.1.0.zip, ca4367c565026ea23bf90180cdd750a1d6adfab36ee19435e91ed740a369a6fc`

Comment (niheaven) [reply bucket/monero-cli.json]: `.*` works, too. But okay, let's use exactly match

Comment (Ash258) [reply bucket/monero-cli.json]: That is nonsense to use greedy `.*` for two characters

**Pull request merged**